src/install-opentofu.sh

```diff
@@ -371,6 +371,21 @@ deb_download_gpg() {
   return "${TOFU_INSTALL_EXIT_CODE_OK}"
 }
 
+# This function sets permissions for the OpenTofu sources list file.
+set_sources_list_permissions() {
```

Comment:
Why is this a separate function? It seems to be used in only one place.

Reply:
I thought it would be better for code readability purposes. I can shift this logic back to where it is being called 👍🏼